assets/src/block-editor/components/preview-menu-item.js

```diff
@@ -221,37 +220,36 @@ class AmpPreviewButton extends Component {
 			isEnabled &&
 			!errorMessages.length &&
 			!isStandardMode && (
-				<Button
+				<PluginPreviewMenuItem
```

Comment:
We'll need to make sure that if `PluginPreviewMenuItem` doesn't exist due to not being on the latest version of WordPress that we short-circuit this function so we don't get an error.